static/app/views/relocation/getStarted.tsx

```diff
@@ -52,18 +52,28 @@ function GetStarted(props: StepProps) {
             minLength={3}
             placeholder="org-slug-1, org-slug-2, ..."
           />
-          <Label>{t('Choose a datacenter region')}</Label>
+          <Label>{t('Choose a datacenter location')}</Label>
           <RegionSelect
             value={regionUrl}
             name="region"
             aria-label="region"
-            placeholder="Select Region"
+            placeholder="Select Location"
             options={regions.map(r => ({label: r.name, value: r.url}))}
             onChange={opt => setRegionUrl(opt.value)}
           />
-          {regionUrl && (
-            <p>{t('This is an important decision and cannot be changed.')}</p>
-          )}
+          <p>
```

Comment:
There seems to be a lot of spacing between this text and the Continue Button. Could we tighten that up?

Reply:
Fixed, new spacing:

![Screenshot 2024-01-23 at 11 58 51](https://github.com/getsentry/sentry/assets/3709945/6815812a-4676-4560-b20a-76647fb26b7b)